pkg/kf/commands/apps/integration_test.go

```diff
@@ -118,20 +117,17 @@ func checkApp(
 	// 2. Tests work even if a domain isn't setup.
 	Logf(t, "hitting echo app to ensure its working...")
 
-	// TODO(#46): Use port 0 so that we don't have to worry about port
-	// collisions.
-	go kf.Proxy(ctx, appName, proxyPort)
-	assert(ctx, t, fmt.Sprintf("http://localhost:%d", proxyPort))
+	go kf.Proxy(ctx, appName)
+	assert(ctx, t, fmt.Sprintf("http://localhost:0"))
```

Comment:
So I don't think this will work. The assertion function will need to know where to hit. 

Port 0 is special in that it will instruct the kernel to just give it an available port when binding. I believe `kf proxy` returns the bound port. You'll need to have the integration test wrapper around `kf proxy` parse the address and return it so that downstream things can hit the address.

Reply:
I will try and fix that.